wagtail/admin/templates/wagtailadmin/userbar/base.html

```diff
@@ -31,7 +31,11 @@
                 {% endfor %}
             </ul>
         </div>
-        <script src="{% versioned_static 'wagtailadmin/js/userbar.js' %}"></script>
     </div>
-</div>
+</template>
+<template id="wagtail-annotation-template">
+    <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/userbar.css' %}">
+</template>
+<wagtail-userbar>Test</wagtail-userbar>
+<script src="{% versioned_static 'wagtailadmin/js/userbar.js' %}"></script>
```

Comment:
The userbar template is used for the userbar, and the annotation template for in-page Axe errors. Both load our stylesheet separately but it’ll only be loaded by the browser once.